### PR TITLE
Chris nickell   new branch

### PR DESCRIPTION
**Pull request by twopercentjazz**
Hi team, I'm late to the party but here is my pull request for my branch. 

I changed index.html to include a hyperlink (to github.com) on the text 'git'

Can someone create a pull request that I can merge?

Thanks!

### CODE REVIEW

**Comment by AmnaKhider** [comment]
Okay!

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1 style="text-align: center;">
               
            
             <h3>It's going to be a good quarter to learn </h3>            
-            <p> Let's 'git' started! </p>
+            <p>Let's '<a href="https://github.com/">git</a>' started! </p>
```
Comment (twopercentjazz) [inline index.html:21]: added a hyperlink

**Comment by AmnaKhider** [comment]
Oh,  now I see your change in the code.

**Comment by AmnaKhider** [comment]
I think now you can delete branch or not yet?

**Comment by firejewels** [comment]
Thanks Everyone!

**Pull request merged**